test-utils/src/lib.rs

```diff
@@ -17,6 +17,28 @@ pub fn bits256_to_h256(b: Bits256) -> H256 {
     H256(b.0)
 }
 
+// Fuel uses compact serialization for signatures following EIP 2098
+// See https://eips.ethereum.org/EIPS/eip-2098
+// |    32 bytes   ||           32 bytes           |
+// [256-bit r value][1-bit v value][255-bit s value]
+pub fn signature_to_compact(signature: &Signature) -> [u8; 64] {
```

Comment:
can we use https://github.com/FuelLabs/fuel-vm/blob/master/fuel-crypto/src/signature.rs ?

Reply:
oh good find

Looks like they have a way to convert from secp256k1::RecoverableSignature https://github.com/FuelLabs/fuel-vm/blob/master/fuel-crypto/src/signature.rs#L146 but I can't figure out how to go from an ethers-rs Signature to a RecoverableSignature...